### PR TITLE
Issue template tweak and Update build to 7

### PR DESCRIPTION
**Pull request by yakov116**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -24,7 +24,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.0.0-alpha6'
+        classpath 'com.android.tools.build:gradle:3.0.0-alpha7'
```
Comment (k0shk0sh) [inline build.gradle]: stash this

File: app/src/main/java/com/fastaccess/ui/modules/main/donation/CheckPurchaseActivity.kt [hunk 2]
```diff
@@ -22,7 +20,7 @@ class CheckPurchaseActivity : Activity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         NotificationSchedulerJobTask.scheduleOneTimeJob(applicationContext)
-        if (AppHelper.isGoogleAvailable(this)) {
+        if (AppHelper.isGoogleAvailable(this) && !AppHelper.isEmulator()) {
```
Comment (k0shk0sh) [inline app/src/main/java/com/fastaccess/ui/modules/main/donation/CheckPurchaseActivity.kt]: stach this

File: app/src/main/java/com/fastaccess/helper/AppHelper.java [hunk 3]
```diff
@@ -136,7 +137,7 @@ public static String getDeviceName() {
         return brand.equalsIgnoreCase(model) ? InputHelper.capitalizeFirstLetter(model) : InputHelper.capitalizeFirstLetter(brand) + " " + model;
     }
 
-    private static boolean isEmulator() {
+    public static boolean isEmulator() {
```
Comment (k0shk0sh) [inline app/src/main/java/com/fastaccess/helper/AppHelper.java]: stash this

**Comment by k0shk0sh** [comment]
@barreeeiroo

**Comment by k0shk0sh** [comment]
@kutsan

**Comment by barreeeiroo** [comment]
It hasn't crashed now :+1:

_Sent from my Bq Aquaris E5 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by yakov116** [comment]
@k0shk0sh please don't close this I need it for testing

_Sent from my Google Nexus 9 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Pull request merged**